shopfloor/services/cluster_picking.py

```diff
@@ -112,7 +112,8 @@ def list_batch(self):
     def _select_a_picking_batch(self, batches):
         # look for in progress + assigned to self first
         candidates = batches.filtered(
-            lambda batch: batch.state == "in_progress" and batch.user_id == self.env.user
+            lambda batch: batch.state == "in_progress"
+            and batch.user_id == self.env.user
```

Comment:
Is it what black did with pre-commit? It doesn't do this on my side

Reply:
Travis was complaining about it (or was it only a warning ?) I may have forced it with black to get the unit tests working. Will revert back to see what happens...